keploy.sh

```diff
@@ -109,7 +119,9 @@ installKeploy (){
         if ! $network_alias docker network ls | grep -q 'keploy-network'; then
             $network_alias docker network create keploy-network
         fi
+    }
 
+    install_docker() {
```

Comment:
why haven't you called add_network in this function? In macos, we still install keploy via docker, so it will be needed in that case. In the future when we add a binary for macos, we can remove the install_docker completely

Reply:
Added add_network in darwin